gcloud-java-storage/src/main/java/com/google/gcloud/storage/StorageImpl.java

```diff
@@ -577,6 +577,68 @@ public URL signUrl(BlobInfo blobInfo, long expiration, SignUrlOption... options)
     }
   }
 
+  @Override
+  public List<BlobInfo> get(BlobInfo blobInfo1, BlobInfo blobInfo2, BlobInfo... blobInfos) {
```

Comment:
Yes I think that we would have a problem with `update`. With the signatures:
```java
update(BlobInfo... , Options...)
update(BlobInfo, BlobInfo...)
```
The call `update(blobInfo)` is ambiguous.